content/version_control/VCS.tex

```diff
@@ -0,0 +1,158 @@
+
```

Comment:
Is this the same than the one on the VCS pull request?

Reply:
It is a different slide set. One separate for VCS and one for Git.

Reply:
I know, what I mean is ... is this the same slide set in #10 ?

Reply:
It is the old slide set. Not the latest one.

Reply:
Alright, then could you remove this one? Otherwise, we'll get merge conflicts.